### ISSUE TITLE
Looks cool / usefull but...

### ISSUE THREAD

**Issue by chrisribe** [olahol/react-ab#13]
Not an issue but on first look it's hard to understand what the react-ab even does (display, etc)...
Even when looking at the code, maybe better doc or simpler API ?

Not a criticism looks usefull just hard to get started.

Keep up the good work.

**Comment by olahol** [comment]
Yeah I should probably add a better description and explain that the only thing `react-ab` does is randomly pick a variation to render, informs you what variation was picked and then pins that variation by storing a cookie, it's useful if you want to setup simple A/B tests with GA and Mixpanel.

**Comment by ryancoughlin** [comment]
This would be awesome. Looking to wire this up with Gatsby.

**Comment by toddheslin** [comment]
@ryancoughlin I was looking for the same thing! Any luck? About to give it a try now.
